src/main/java/com/auctionapp/api/configuration/SecurityConfiguration.java

```diff
@@ -27,6 +27,7 @@
 
     private final UserDetailsService userDetailsService;
     private final JwtAuthenticationFilter jwtAuthenticationFilter;
+    public static final String HOST = "bidba-api.herokuapp.com";
```

Comment:
This is still code, I was thinking more along the lines of having this in a .yml file or similar.